Identity/Dockerfile

```diff
@@ -1,7 +1,8 @@
 ﻿FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
 USER $APP_UID
 WORKDIR /app
-EXPOSE 8080
+EXPOSE 8081
+ENV ASPNETCORE_URLS=http://*:8081
```

Comment:
Why is this needed? You can just specify a port binding when running a container, e.g. if you want to reach the container on port 8081 and the web server inside the container listens on 8080, you just specify `-p 8081:8080`, [docs](https://docs.docker.com/network/#published-ports)

Reply:
you should do that as well [here](https://github.com/BenasB/vu-psk/blob/54955f13b026c821fe8de45f3299a3ee6236f9c7/Identity/docker-compose.yaml#L7)

Reply:
Changed this